### ISSUE TITLE
package with resource from plastanium belt floatig without belt

### ISSUE THREAD

**Issue by DEpObORnik** [Anuken/Mindustry#9974]
### Platforms

Windows

### Build

steam build 146

### Issue

![image](https://github.com/Anuken/Mindustry/assets/111637832/8d8a4f87-5840-4501-8476-a2512a8db047)
this thorium package floating without plast belt. capacity is full(10). if i click on it, i cant pick it up. im the host, this happend to me while i was playing campaign. i have steam version.

### Steps to reproduce

idk, i just saw that this thorium pack wasnt moving and other packages couldnt go further.

### Mods used

scheme size, Extended Ui

### Save file

campaign, 54 sector

### Submission

- [X] I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.
- [X] I have searched the closed and open issues to make sure that this problem has not already been reported.
- [X] I am not using Foo's Client, and have made sure the bug is not caused by mods I have installed.

**Comment by DEpObORnik** [comment]
aaaaa i forgor to add game data and now i cant cuz file size i too big(68 mb)

**Comment by secondvince** [comment]
The real problem is that if you disable a plast conveyor, its animation stops.
The floating package is the part of the next comnveyor, not the deconstructed one.

I made a playground map to reproduce this:
[bug.zip](https://github.com/user-attachments/files/15924563/bug.zip)
Its not a zip, i renaned a map file for uploading.

To reproduce, you should connect a belt to the processor, and the proc will disable it.
If the disabled conveyor has some content, the contents image wont move even if it is not above the conveyor.
If the disabled conveyor has no content, you can drop something to a disabled conveyor, its new content wont appear.

To sum up disabled conveyors appearance does not update and this ends up in very strange situations.

**Comment by Matita008** [comment]
@DEpObORnik close this, it's intended behavior

**Comment by mahdoosh1** [comment]
> @DEpObORnik close this, it's intended behavior 

Are you sure?
